src/main/java/com/pi4j/example/components/ADS1115.java

```diff
@@ -306,75 +312,66 @@ public void stopFastContinuousReading() {
     }
 
     /**
-     * start continuous reading
+     * start slow continuous reading on custom channel
      *
      * @param threshold     threshold for trigger new value change event (+- digit)
      * @param readFrequency read frequency to get new value from device, must be lower than 1/2
      *                      sampling rate of device
      */
-    public void startSlowContinuousReading(double threshold, int readFrequency) {
+    public void startSlowContinuousReading(int channel, double threshold, int readFrequency) {
+        logDebug("Start slow continuous reading chanel " + channel);
         //only start continuous Reading if it is not already running because of other component
-        if(!continuousReadingActive){
-            slowReadContinuousValue(threshold, readFrequency);
+        if (!continuousReadingActive) {
             continuousReadingActive = true;
+            slowReadContinuousValue(threshold, readFrequency);
         }
+        continuousReadingActiveChannel[channel] = true;
     }
 
     /**
-     * stops continuous reading
+     * stops continious reading
```

Comment:
continuous would be right. make sure you set it everywhere by searching for the wrong